include/rocksdb/iterator.h

```diff
@@ -1,3 +1,17 @@
+// Copyright (C) 2023 Speedb Ltd. All rights reserved.
```

Comment:
This feels excessive for adding a function hardcoded to return true. It's line 1 of a common public header file. If we did this for every contributor the license lines would be thousands of lines in every file.

Reply:
Removed it

Reply:
> How do you suggest to do it differently?

We would probably pass on most (not all) PRs for which copyright changes are needed. I don't really have a suggestion on how to do it differently